config/config-default/migrate_plus.migration.term_subject_matter.yml

```diff
@@ -0,0 +1,38 @@
+uuid: aea7b239-7b78-48eb-82b7-8a9614da67be
```

Comment:
This file was lovingly hand crafted (without this uuid) and imported via `drush cim`.  That gets the config into drupal.  

After that, config is exported with `drush cex`. On the export the file is overwritten with Drupal's internal config which adds the uuid.  This uuid forces configuration to be specific to an environment.  This is why all environments, (including locals) must start from the same DB otherwise these uuids won't be compatible and the config won't import.

Fun fact:  If you know what you're doing and you have a config file you know is compatible with the rest of your site config, you can strip the uuid line of the incompatible file and import it anyway, (you'll get errors if it doesn't mesh with current configuration).  This is not recommended, but it's the easiest way I know to create NEW migration configuration.

Reply:
looks like we're not storing the old nsmi uuid.  in a future migration of an update, if nsmi changes the label of "Attorney Fees" to something else, how do we map it to the old term?